packages/embarkjs-ipfs/src/index.js

```diff
@@ -37,9 +37,9 @@ __embarkIPFS.isAvailable = function () {
     if (!this._ipfsConnection) {
       return resolve(false);
     }
-    this._ipfsConnection.id()
-      .then((id) => {
-        resolve(Boolean(id));
+    this._ipfsConnection.version()
```

Comment:
Wondering if that breaks with older versions of IPFS or something? Or maybe it doesn't matter because they would be deprecated?

Reply:
I hazily remember that there was a discussion quite some time ago re: why we use `id()` instead of `version()`. I searched around in changelogs and source code for go-ipfs and ipfs-api (renamed to ipfs-http-client) but I couldn't find anything that stood out to me as a good reason to prefer id over version.

Reply:
> Wondering if that breaks with older versions of IPFS or something? Or maybe it doesn't matter because they would be deprecated?

Was wondering the same, but I think `version()` has been there all along.

Reply:
I didn't encounter any problems with these changes in place when manually testing for the 4.1.0 release. 👍